weblink/_internal/Server.hx

```diff
@@ -96,16 +194,44 @@ class Server extends SocketServer {
 		}
 	}
 
-	public function update(blocking:Bool = true) {
-		do {
-			@:privateAccess MainLoop.tick(); // for timers
-			loop.run(NoWait);
-		} while (running && blocking);
-	}
+	@:deprecated("Updates are now done in a background thread. You should remove this call.")
+	public function update(blocking:Bool = true) {}
```

Comment:
I'd revert this change as previous weblink servers may still reley on this, and we can deprecate it later.

Reply:
I believe I can make this method block until the server is closed, but I don't think it should tick the loops: running libuv from two threads at once is a sure way to crash the app with some form of "assertion failed".

Would that behavior be OK with you?

Reply:
I see your point, okay this is fine then.